src/test/java/net/snowflake/client/jdbc/PreparedStatement2LatestIT.java

```diff
@@ -216,7 +216,7 @@ public void testExecuteLargeBatch() throws SQLException {
 
   @Test
   public void testRemoveExtraDescribeCalls() throws SQLException {
-    Connection connection = getConnection();
+    Connection connection = init();
```

Comment:
Forgot that for tests that run on both Arrow and Json (like PreparedStatement2LatestIT), we should always use init() instead of getConnection() because the setup logic for init() ensures we make the switch from json to arrow for the arrow extension IT tests. Fixed those too.